doc/configuration.md

```diff
@@ -4,15 +4,17 @@ VS Code settings to configure Joyride.
 
 There is no configuration. 😀
 
-There are some conventions, though. Joyride looks for workspace scripts in:
+There are some conventions, though. Joyride looks for scripts in:
 
-```
-.joyride/scripts/**/*.cljs
-```
+* User scripts: `<user home>/.config/joyride/scripts/**/*.cljs`
+* Workspace scripts: `.joyride/scripts/**/*.cljs`
```

Comment:
Is it an idea to only include top level scripts so library scripts can live in their own directory and won't be shown?

Reply:
I think that it scales better to let people just organize their scripts folder as they like. Since the menu has fuzzy search, it helps quickly find a specific script when organized in subfolders. A `.menuignore` file in the `.joyride` dir, or the kind of config file we discussed elsewhere can help the user take care of the menu, beyond the `z-` prefix I use in the examples folder.

An ignore file is extra easy to implement as we can just pick the patterns there and add to the `excludePatterns` argument where we glob for files. Also quite familiar for the users, I think.

Reply:
I recently saw your `z_lib.cljs` workaround and that could have been solved by:

```
my_script.cljs
my_lib/script.cljs
```

I'm not sure why I would ever want to have scripts from subdirectories appear in my menu and now we still have the change to change that.

I don't think we need the .ignore file if we have the explicit `index.edn` which gives more control over the layout and keybindings.

Reply:
Either way, it's not a dealbreaker for me, let's merge.